templates/admin/programmatic_foundation/measure/link.html.twig

```diff
@@ -0,0 +1,4 @@
+<a href="#" target="_blank" class="btn btn-sm btn-default" title="Voir la mesure">
+    <i class="fa fa-eye" aria-hidden="true"></i>
+    Voir la mesure (dummy)
```

Comment:
(dummy) ?

Reply:
The link will be implemented in the next MR, I just wanted to notify that's (for now) a broken link

Reply:
I'll remove that file btw because it's not part of that PR